packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -177,6 +179,14 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         return (await this.getInitialSnapshotDetails()).isRootDataStore;
     }
 
+    public get loadContainerCopyFn(): (
```

Comment:
why do we need this. i don't think it should exist. datastores shouldn't be creating container copies at will

Reply:
This was to pipe down the callback into `scheduler.ts` (above).  It's strictly less functionality than provided by the loader that gets piped down as well, though it's possible we don't even want this much by default...

I can remove it for now, but the scheduler needs to be accounted for somehow.  @ChumpChief has #4413 right now which involves potentially removing the container creation part of the scheduler, so depending on the decisions there it may not be an issue or we may potentially need to add this back

Reply:
Jk looked into this again, the paired ODSP url resolver and driver use it but the resolved url is passed around as an IResolvedUrl and cast to IOdspResolvedUrl in the driver...  The url resolver certainly doesn't care about that, but there isn't currently a good way to communicate that to the driver on the types used in the interface.  Going to add a client type -> header translation here as a temporary workaround until that's resolved